scripts/test-uma-ODRL-policy.ts

```diff
@@ -0,0 +1,17 @@
+const endpoint = 'http://localhost:4000/uma/policies'
+const client1 = 'https://pod.woutslabbinck.com/profile/card#me';
+const client2 = 'https://pod.example.com/profile/card#me';
+
+
+async function main() {
```

Comment:
could you also add a test where there is no authorization provided? saying that you expect an HTTP 4xx message